kernel/kernel-api/src/main/java/io/delta/kernel/parquet/ParquetFooter.java

```diff
@@ -0,0 +1,17 @@
+package io.delta.kernel.parquet;
+
+import io.delta.kernel.types.StructType;
+
+/**
+ * Class representing a subset of information from the Parquet file footer.
+ */
+public class ParquetFooter
```

Comment:
Lets remove this. We can add this back when id based column mapping is supported.